oca_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+base_vat_sanitized https://github.com/OCA/partner-contact.git
+account_tax_unece https://github.com/onesteinbv/community-data-files 11_mig_account_tax_unece
+product_uom_unece https://github.com/OCA/community-data-files
```

Comment:
All the modules in the dependencies are merged. You can rewrite the `oca_dependencies.txt` using the definitive list of repositories.

Reply:
Ok done, thanks @astirpe